packages/firebase_auth/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.14.0+7
+* Fix NoSuchMethodError exception in `reauthenticateWithCredential`
```

Comment:
```suggestion

* Fix NoSuchMethodError exception in `reauthenticateWithCredential`
```

Reply:
Maybe I should not comment here, but `NoSuchMethodError` exception is somewhat  contradictory as [exceptions and errors are not the same in Dart](https://stackoverflow.com/a/58361954/6509751).